packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method.jsx

```diff
@@ -4,13 +4,12 @@ import { useStores } from 'Stores';
 import { localize } from 'Components/i18next';
 import { DesktopWrapper, MobileWrapper } from '@deriv/components';
 import AddPaymentMethodForm from './add-payment-method-form.jsx';
-import CancelAddPaymentMethodModal from './cancel-add-payment-method-modal.jsx';
 import PageReturn from 'Components/page-return/page-return.jsx';
 import PropTypes from 'prop-types';
 import SelectPaymentMethod from './select-payment-method.jsx';
 
 const AddPaymentMethod = ({ formik_ref, should_show_page_return = true, should_show_separated_footer }) => {
-    const { my_ads_store, my_profile_store } = useStores();
+    const { my_ads_store, my_profile_store, general_store } = useStores();
 
     React.useEffect(() => {
         my_profile_store.setIsCancelAddPaymentMethodModalOpen(false);
```

Comment:
shouldn't we set it using general_store.hideModal?

Reply:
Ooof we don't need this anymore